assets/js/components/HostDetails/HostClusterAgentIpSummary.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ListView from '@components/ListView';
+import HealthIcon from '@components/Health/HealthIcon';
+import Tooltip from '@components/Tooltip';
+import ClusterLink from '@components/ClusterLink';
+
+import { chunk } from 'lodash';
+
+function HostClusterAgentIpSummary({ agentVersion, cluster, ipAddresses }) {
+  const prepareTooltipContent = (ipList) => {
```

Comment:
Thanks for the explanation, yeah moving it outside!

Abut the naming... I went with "ipList" instead of "ipAddresses" because, well, in the HostSummary component, we use "ipAddresses" specifically. But for the other functions, they're a bit more independent, and I wanted a simple and clear name to indicate that we're working with a list of IP addresses. "ipList" just seemed more straightforward and less specialized than "ipAddresses, but both works here fine what do you think ?